Assets/Scripts/Door.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class Door : MonoBehaviour
+{
+    BoxCollider2D box;
+    
+    GameObject[] EnemyBandits;
+
+    int currentScene;
+
+    private void Start()
+    {
+        currentScene = SceneManager.GetActiveScene().buildIndex;
+        EnemyBandits = GameObject.FindGameObjectsWithTag("EnemyBandit");        
+    }
+
+    private void Awake()
+    {
+        box = GetComponent<BoxCollider2D>();
+    }
+
+    void OnTriggerStay2D(Collider2D other)
+    {
+        if ((Input.GetKey(KeyCode.UpArrow) || Input.GetKey(KeyCode.W)) && other.tag == "Player")
```

Comment:
Use the input manager (`Input.GetButton`  or `Input.GetAxis` and related methods)

Reply:
i.e., don't use input.getkey